src/components/FlashWord/index.jsx

```diff
@@ -17,8 +17,18 @@ export default function FlashWord({ corrects, word }) {
   }, [word, corrects]);
 
   return (
-    <p className={`${styles.word} ${hide ? "" : styles.hidden}`}>
-      {word.toUpperCase()}
+    <p className={styles.word}>
+      {word.split("").map((letter, idx) => {
+        let style = {
+          transition: "opacity .5s ease-in-out",
+          "transition-delay": idx / 10 + "s",
+        };
+        return (
+          <span key={idx} style={style} className={hide ? "" : styles.hidden}>
```

Comment:
Two things:

1. I'm not a fan of inline styling here just because styling is done with the module everywhere else. That said, I'm not sure how to code the "delay" without doing it this way. So, i'm ok with it.
2. Why have style as a variable when the object could be put in the `span` directly? If you do want to keep it as a variable, perhaps `const` is more appropriate than `let`?